decidim-meetings/app/controllers/decidim/meetings/admin/meeting_closes_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Decidim
+  module Meetings
+    module Admin
+      # This controller allows an admin to manage meetings from a Participatory Process
+      class MeetingClosesController < Admin::ApplicationController
+        helper ResourceHelper
+
+        def edit
```

Comment:
The thing is that there aren't any close meeting model, it's just updating an existing model. I guess I can add edit/update too but they'll do the same as new/create.